discretize/TreeMesh.py

```diff
@@ -728,6 +728,15 @@ def gridN(self):
         R = self._deflationMatrix('N', withHanging=False)
         return R.T * self._gridN + np.repeat([self.x0],self.nN,axis=0)
 
+    @property
+    def h_gridded(self):
```

Comment:
could you please add a docstring to this?

Reply:
Is this done through simpeg or discretize repository or both?

Reply:
this is done in the code
```
"""
Docstring for h_gridded
"""
```